src/lib/includes/kfs_common.h

```diff
@@ -8,111 +8,19 @@
 #include <stdio.h>
 #include <zconf.h>
 #include <dirent.h>
+#include <assert.h>
```

Comment:
why it needed?

Reply:
for str_concat (kfs_common line 4 and 7)